doc/source/yt4differences.rst

```diff
@@ -92,6 +92,45 @@ arbitrary_grid = ds.arbitrary_grid([0.0, 0.0, 0.0], [5, 5, 5],dims=[10, 10, 10])
 density = arbitrary_grid[('gas', 'density')]
 print(density)
 ```
+Off-Axis Projection for SPH Data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The current `off_axis_projection` function will now support SPH projection plots.
+
+The following code generates an off-axis projection of an example dataset with a given
+normal vector, resolution, center, width, and smoothing quantitiy. 
+
+.. code-block:: python
+    import yt
+
+    ds = yt.load('GadgetDiskGalaxy/snapshot_200.hdf5')
+
+    normal_vector = [0., 1., 0.]
+
+    resolution = (256, 256)
+
+    left_edge = ds.domain_left_edge
+
+    right_edge = ds.domain_right_edge
+
+    center = [(left_edge[0] + right_edge[0])/2,
+              (left_edge[1] + right_edge[1])/2,
+              (left_edge[2] + right_edge[2])/2]
+
+    width = [(right_edge[0] - left_edge[0]),
+             (right_edge[1] - left_edge[1]),
+             (right_edge[2] - left_edge[2])]  
+
+    smoothing_quantity = 'particle_mass'  
+
+    buf = OffAP.off_axis_projection(ds,
+                                    center,
+                                    normal_vector,
+                                    width,
+                                    resolution,
+                                    smoothing_quantity
+                                    )
```

Comment:
Can you make this example use `OffAxisProjectionPlot` instead? I think that's surfaced much more clearly in our API than `off_axis_projection`.

Please also make sure that this example runs correctly when you copy/paste the code into a script or a jupyter notebook.

Reply:
Alright I'll read up on `OffAxisProjectionPlot`

Reply:
The example now uses `OffAxisProjectionPlot`